### DISCUSSION TITLE
What is the password of the root account in docker image  v5.3.2

### DISCUSSION THREAD

**Discussion by liyy1** [emqx/emqx#12254] [Q&A] [answered]
What is the password of the root account in docker image
image version: emqx:5.3.2

**Comment by id** [comment, marked as answer]
@liyy1 if you want to get shell as root in emqx docker container, you can do this (assuming your container's name is emqx):

```
docker exec -it -u root emqx bash
```

**Reply by liyy1** [reply]
Thanks for your help
